sdcm/utils/decorators.py

```diff
@@ -155,6 +158,96 @@ def wrapped(*args, **kwargs):
     return wrapped
 
 
+class LatencyCalculatorMixedResult(GenericResultTable):
+    class Meta:
+        name = ""  # to be set by the decorator
```

Comment:
if so, why we need to define those here ?

I thought that we would define the exact tables, and then use it where need.

now it's seem most of the things are dynamic but the columns (which can be dynamic as well)

Reply:
yes, it's dynamic as latency_decorator is decorating different methods - and each method is a separate measurement (thus requires different table name).
Columns are static - (as per definition which can change in future)